File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    
    int count = 0;
    float o;
    do
    {
        o = get_float("Change owend: $");
    }
    while (o < 0);
    float c = round(o * 100);
    
    do
    {
        if (c >= 25)
        {
            c -= 25;
            count++;
        }
        else if (c >= 10 && c < 25)
        {
            c -= 10;
            count++;
        }
        else if (c >= 5 && c < 10)
        {
            c -= 5;
            count++;
        }
        else if (c >= 1 && c < 5)
        {
            c -= 1;
            count++;
        }
    }
    while (c != 0);
    printf("Total coins: %i\n", count);
}

File: cash.py
from cs50 import get_float

while True:
    change = get_float("Change owned: $")
    if change > 0:
        break

coins = 0
change = round(change * 100)
while change > 0:
    if change >= 25:
        change -= 25
    elif change >= 10:
        change -= 10
    elif change >= 5:
        change -= 5
    else:
        change -= 1
    coins += 1
print(f"{coins}")